libp2p/ENR.h

```diff
@@ -64,6 +68,7 @@ class ENR
     bytes content() const;
     size_t contentRlpListItemCount() const { return m_keyValuePairs.size() * 2 + 1; }
     void streamContent(RLPStream& _s) const;
+    Host* h;
```

Comment:
This is not correct (`ENR` can belong to some peer, not to `Host`) and not needed, just use `streamRLP` of this object in `encodeEnr`